tests/scancode/test_ignore_files.py

```diff
@@ -24,6 +24,13 @@
 
 from unittest import TestCase
 from commoncode.ignore import is_ignored
+from scancode.cli import resource_paths
+from commoncode.testcase import FileDrivenTesting
+from os import path
+
+test_env = FileDrivenTesting()
```

Comment:
This pattern of using file-driven testing should not be used except in test_cli.py. Subclass instead FileBasedTesting rather than the stdlib TestCase anywhere else.